PasswordManager/PasswordManager.xcodeproj/project.pbxproj

```diff
@@ -7,15 +7,23 @@
 	objects = {
```

Comment:
Since this file is automatically generated by Xcode, should we probably not be committing this from individual branches? Actually, same with the other files. I noticed there are over 3k files changed here...

Reply:
I think that maybe we can take this out for now and put it back in to the project if we need it